packages/ui/src/composites/wui-list-account/index.ts

```diff
@@ -0,0 +1,131 @@
+import { html, LitElement } from 'lit'
+import { property } from 'lit/decorators.js'
+import '../../components/wui-text/index.js'
+import '../../components/wui-image/index.js'
+import '../../layout/wui-flex/index.js'
+import { elementStyles, resetStyles } from '../../utils/ThemeUtil.js'
+import { customElement } from '../../utils/WebComponentsUtil.js'
+import styles from './styles.js'
+import { UiHelperUtil } from '../../utils/UiHelperUtil.js'
+import { W3mFrameRpcConstants } from '@web3modal/wallet'
+import {
+  AccountController,
+  BlockchainApiController,
+  NetworkController,
+  StorageUtil
+} from '@web3modal/core'
+
+@customElement('wui-list-account')
+export class WuiListAccount extends LitElement {
+  public static override styles = [resetStyles, elementStyles, styles]
+
+  // -- State & Properties -------------------------------- //
+  @property() public accountAddress = ''
+
+  @property() public accountType = ''
+
+  private connectedConnector = StorageUtil.getConnectedConnector()
+
+  private labels = AccountController.state.addressLabels
+
+  private caipNetwork = NetworkController.state.caipNetwork
+
+  private balance = 0
+
+  private fetchingBalance = true
+
+  private shouldShowIcon = false
+
+  @property({ type: Boolean }) public selected = false
+
+  @property({ type: Function }) public onSelect?: (
+    { address, type }: { address: string; type: string },
+    selected: boolean
+  ) => void
+
+  public override connectedCallback() {
+    super.connectedCallback()
+    BlockchainApiController.getBalance(this.accountAddress, this.caipNetwork?.id).then(response => {
+      let total = this.balance
+      if (response.balances.length > 0) {
+        total = response.balances.reduce((acc, balance) => acc + (balance?.value || 0), 0)
+      }
+      this.balance = total
+      this.fetchingBalance = false
+      this.requestUpdate()
```

Comment:
same as before I'd use state vars

Reply:
updated it but dangerjs complains https://github.com/WalletConnect/web3modal/pull/2286#issuecomment-2208322667